internal/terraform/resource_test.go

```diff
@@ -19,16 +19,67 @@ import (
 	"github.com/terraform-docs/terraform-docs/internal/types"
 )
 
-func TestResourceFullType(t *testing.T) {
+func TestResourceSpec(t *testing.T) {
 	assert := assert.New(t)
 	resource := Resource{
 		Type:           "private_key",
+		Name:           "baz",
 		ProviderName:   "tls",
 		ProviderSource: "hashicorp/tls",
 		Mode:           "managed",
 		Version:        types.String("latest"),
 	}
-	assert.Equal("tls_private_key", resource.FullType())
+	assert.Equal("tls_private_key.baz", resource.Spec())
+}
+
+func TestResourceMode(t *testing.T) {
+	tests := []struct {
+		name          string
+		resource      Resource
+		expectValue   string
+		expectDefault bool
```

Comment:
This seems to be unused. If it was intended to be used please double check down below otherwise remove it.

Reply:
you're right, isn't used, removed!